src/generic.YAML-tmLanguage

```diff
@@ -0,0 +1,39 @@
+repository:
+    generic:
+        patterns:
+            -
+                begin: '(?=\<(?:[A-Z_]|\*|in|out))'

```

Comment:
It might be worth matching the `:` here. It's referenced here https://github.com/nishtahir/language-kotlin/blob/e2b759590ff405dc86f2b8c39ff1dfa4848dae71/src/keywords.YAML-tmLanguage#L52-L54 as a declaration punctuation. Might be worth adding that for consistency